### DISCUSSION TITLE
Possibility to open Forecast tab without it executing simulations

### DISCUSSION THREAD

**Discussion by devyous** [nbrown02/FlowViz#92] [Q&A] [answered]
Hi all, is it possible to alter this fantastic project to not having it immediately executing simulations when opening the Forecast tab?

I’ve changed the language of PowerBI Desktop to English-US, because earlier the page freezes, but even with this change I’m not able to change the Area Path before PowerBI Desktop crashing.

What’s interesting is that PowerBI Desktop crashes with a WebView2 error.

Maybe this error helps us with this troublesome page/tab :P

**Comment by nbrown02** [comment]
It could technically be done I guess :) the problem is the already large complexity that exists with bookmarks, dark mode, etc in the report already. So this won't be something I'll look to do - but happy for others to fork and do :)

**Comment by devyous** [comment, marked as answer]
I'm helped with the alternative  [Area Path ](https://github.com/nbrown02/FlowViz/raw/main/Screenshots/FlowViz%20(Project)%20-%20Area%20Path%20Filter.pbit) or [Team](https://github.com/nbrown02/FlowViz/raw/main/Screenshots/FlowViz%20(Project)%20-%20Team%20Filter.pbit) templates for now, thanks.
